src/main/java/graphql/parser/Parser.java

```diff
@@ -133,8 +176,24 @@ public void syntaxError(Recognizer<?, ?> recognizer, Object offendingSymbol, int
      * @param multiSourceReader the source of the query document
      *
      * @return a new GraphqlAntlrToLanguage instance
+     *
+     * @deprecated - really should use {@link #getAntlrToLanguage(CommonTokenStream, MultiSourceReader, Boolean)}
      */
+    @Deprecated
     protected GraphqlAntlrToLanguage getAntlrToLanguage(CommonTokenStream tokens, MultiSourceReader multiSourceReader) {
```

Comment:
This was left to preserve old behavior - if some one overrides this then they can.